library/core/src/slice/iter.rs

```diff
@@ -1651,10 +1656,10 @@ impl<'a, T> Iterator for ChunksMut<'a, T> {
             None
         } else {
             let sz = cmp::min(self.v.len(), self.chunk_size);
-            // SAFETY: sz cannot exceed the slice length based on the calculation above
+            // SAFETY: This type ensures that any split_at_mut on self.v is valid.
```

Comment:
I think `This type` is confusing if one doesn't have context. We should refer to it as something like

> The self.v contract [...]

That makes the safety section on the field a sort of reference point.